### PR TITLE
dibs: I will implement exercise etl

### PR DESCRIPTION
**Pull request by dvberkel**
Work in progress

### CODE REVIEW

**Comment by dvberkel** [comment]
This still needs some work.

Besides registering it also needs a better printer

**Comment by dvberkel** [comment]
This can be reviewed

**Comment by stevejb71** [comment]
Sure, I'll take a look

File: exercises/etl/test.ml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+open Etl
+
+
+let ae exp got _test_ctxt =
+  let sexpr_of l = List.sexp_of_t (Tuple2.sexp_of_t Char.sexp_of_t Int.sexp_of_t) l in
+  let printer l = Sexp.to_string (sexpr_of l) in
+  assert_equal exp got ~printer
+
+let tests =
+  ["transform one character">::
```
Comment (stevejb71) [inline exercises/etl/test.ml]: The description isn't quite followed - the output map should be from lower case letters:
"... It also stores the letters in lower-case regardless of the case of the input letters ..."

Comment (dvberkel) [inline exercises/etl/test.ml]: Nice one! I did not spot it. I will change it according to the readme

**Comment by dvberkel** [comment]
@stevejb71 I changed the test and the example implementation to match the README.

**Pull request merged**